<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kate Pdf</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="MyGraphicsView" name="graphicsView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_file"/>
    <addaction name="actionRemove_file"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_to_image"/>
    <addaction name="actionLeftRotate"/>
    <addaction name="actionRightRotate"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExport_to_PDF"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Session"/>
    <addaction name="actionLoad_file"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit_Program"/>
   </widget>
   <widget class="QMenu" name="menuMove_Item">
    <property name="title">
     <string>Move Item</string>
    </property>
    <addaction name="actionUpItem"/>
    <addaction name="actionDownItem"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_PDF_converter"/>
    <addaction name="actionAbout_Qt_5"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMove_Item"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Session"/>
   <addaction name="actionLoad_file"/>
   <addaction name="actionExit_Program"/>
   <addaction name="actionSaveFile"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_file"/>
   <addaction name="actionRemove_file"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionExport_to_PDF"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_to_image"/>
   <addaction name="actionLeftRotate"/>
   <addaction name="actionRightRotate"/>
   <addaction name="actionAbout_PDF_converter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="moveToolBar">
   <property name="windowTitle">
    <string>Move toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUpItem"/>
   <addaction name="actionDownItem"/>
  </widget>
  <action name="actionAdd_file">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/add-image.png</normaloff>:/icons/add-image.png</iconset>
   </property>
   <property name="text">
    <string>Add images</string>
   </property>
  </action>
  <action name="actionRemove_file">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/remove-image.png</normaloff>:/icons/remove-image.png</iconset>
   </property>
   <property name="text">
    <string>Remove selected image</string>
   </property>
   <property name="toolTip">
    <string>Remove selecte image</string>
   </property>
  </action>
  <action name="actionExport_to_PDF">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/convert to pdf.png</normaloff>:/icons/convert to pdf.png</iconset>
   </property>
   <property name="text">
    <string>Export to PDF</string>
   </property>
  </action>
  <action name="actionUpItem">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/up-arrow.png</normaloff>:/icons/up-arrow.png</iconset>
   </property>
   <property name="text">
    <string>Up Item</string>
   </property>
  </action>
  <action name="actionDownItem">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/down-arrow.png</normaloff>:/icons/down-arrow.png</iconset>
   </property>
   <property name="text">
    <string>Down Item</string>
   </property>
  </action>
  <action name="actionExit_Program">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit Program</string>
   </property>
   <property name="toolTip">
    <string>Exit Program</string>
   </property>
  </action>
  <action name="actionAbout_PDF_converter">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>About PDF converter</string>
   </property>
  </action>
  <action name="actionAbout_Qt_5">
   <property name="text">
    <string>About Qt 5</string>
   </property>
  </action>
  <action name="actionZoom_to_image">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/adjustSize.png</normaloff>:/icons/adjustSize.png</iconset>
   </property>
   <property name="text">
    <string>Expand zoom</string>
   </property>
  </action>
  <action name="actionLeftRotate">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/left.png</normaloff>:/icons/left.png</iconset>
   </property>
   <property name="text">
    <string>leftRotate</string>
   </property>
   <property name="toolTip">
    <string>Left rotation</string>
   </property>
  </action>
  <action name="actionRightRotate">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/right.png</normaloff>:/icons/right.png</iconset>
   </property>
   <property name="text">
    <string>rightRotate</string>
   </property>
   <property name="toolTip">
    <string>Right rotation</string>
   </property>
  </action>
  <action name="actionPrecision_Rotation_Angle">
   <property name="text">
    <string>Precision Rotation Angle</string>
   </property>
  </action>
  <action name="actionPage_Orientation">
   <property name="text">
    <string>Page Orientation</string>
   </property>
  </action>
  <action name="actionKind_Of_Render">
   <property name="text">
    <string>Kind Of Render</string>
   </property>
  </action>
  <action name="actionPDF_resolution_in_DPI">
   <property name="text">
    <string>PDF resolution in DPI</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save session</string>
   </property>
  </action>
  <action name="actionLoad_file">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open session</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionNew_Session">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/newFile.png</normaloff>:/icons/newFile.png</iconset>
   </property>
   <property name="text">
    <string>New Session</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
